### ISSUE TITLE
full_stack_example for 6.0.0 ?

### ISSUE THREAD

**Issue by bramamad** [elastic/examples#193]
full_stack_example is a great thing for developing and testing. Thanks a lot.

But with 6.0.0 there are several changes: default userid/pw for elasticsearch, setup of templates, indexes, ...

Do you plan an upgrade for 6.0.0 ?

**Comment by gingerwizard** [comment]
Yes we plan to update all examples. We are slowly working through them. We will also add Logstash and several other systems to the full stack example in due course.

**Comment by ynux** [comment]
Thanks for the Watcher History Dashboard. For 6, this is something that I came across, in case it helps: In the "Most Frequent Watches Executing Actions" visualization, the search now has to be "result.condition.met: true", not "result.condition.met: 1"

**Comment by chrisgaeth** [comment]
Any updates on this as it's been nearly 5 months?
